Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -1432,12 +1432,354 @@ public void When_string_containment_is_asserted_and_actual_value_is_null_then_it
             // Assert
             act.Should().Throw<XunitException>().WithMessage(
 #if NETCOREAPP1_1
-                 "Expected string <null> to contain \"XYZ\" because that is required.");
+                "Expected string <null> to contain \"XYZ\" because that is required.");
 #else
                 "Expected someString <null> to contain \"XYZ\" because that is required.");
 #endif
         }
 
+        #region Exactly
+
+        [Fact]
+        public void When_string_containment_once_is_asserted_and_actual_value_does_not_contain_the_expected_string_it_should_throw()
+        {
+            // Arrange
+            string actual = "ABCDEF";
+            string expectedSubstring = "XYS";
+
+            // Act
+            Action act = () => actual.Should().Contain(expectedSubstring, Exactly.Once(), "that is {0}", "required");
+
+            // Assert
+            act.Should().Throw<XunitException>()
+                .WithMessage("Expected * \"ABCDEF\" to contain \"XYS\" exactly 1 time because that is required, but found 0 times.");
```

Comment:
Something sounds odd to me about the part `but found 0 times.`
Brainstorming some alternatives:
*  `but found it 0 times.`
*  `but found 0 occurences.`

Reply:
I would stick to _times_. If you would like to use the word _occurrence_, the entire message should change.

Reply:
@dennisdoomen So you would change the current wording to "but found it 0 times"?

Reply:
Yep